stl/inc/filesystem

```diff
@@ -2469,6 +2469,14 @@ namespace filesystem {
             return _Path >= _Rhs._Path;
         }
 
+        // [fs.dir.entry.io], inserter
+        template <class _Elem, class _Traits>
+        friend _STD basic_ostream<_Elem, _Traits>& operator<<(
+            _STD basic_ostream<_Elem, _Traits>& _Ostr, const directory_entry& _Entry) {

```

Comment:
Following `operator <<(basic_ostream, path)` for now. I think I need maintainer input on VSO-570323.

Reply:
That bug is "`/Za` rejects `filesystem::path`'s friend streaming operator" and it's still active. If the workaround is also necessary here, it should have the same TRANSITION comment. (I am hopeful that this will become moot in vNext if/when we can remove `/Za` as being effectively superseded by `/permissive-`.)

Reply:
FWIW, this looks good modulo the missing `// TRANSITION, VSO-570323`.